petastorm/unischema.py

```diff
@@ -401,19 +401,27 @@ def insert_explicit_nulls(unischema, row_dict):
                 raise ValueError('Field {} is not found in the row_dict, but is not nullable.'.format(field_name))
 
 
+def _fullmatch(regex, string, flags=0):
+    """Emulate python-3.4 re.fullmatch()."""
+    m = re.match(regex, string, flags=flags)
+    if m and m.span() == (0, len(string)):
```

Comment:
Could we use explicit parentheses here, i.e. `if m and (m.span() == (0, len(string))):`

Reply:
Done